index.html

```diff
@@ -1978,12 +1978,14 @@ <h3>Terms of Use</h3>
         <h3>Evidence</h3>
 
         <p>
-Evidence is used by an <a>issuer</a> to determine whether or not to issue
-a <a>credential</a>. For example, an issuer might check physical documentation
-provided by the <a>subject</a> or might perform a set of background checks
-before issuing the credential. In certain scenarios, this information is useful
-to the <a>verifier</a> when determining the risk associated with relying on a
-given credential.
+Evidence is used by an <a>issuer</a> to inform the <a>verifier</a> of the basis
+for the <a>verifiable credential</a>. This could be used by the <a>verifier</a>
+to establish the confidence with which it relies on the claims in the
+<a>verifiable credential</a>. For example, an issuer might check physical 
```

Comment:
Missing &lt;a>&lt;/a> around issuer.

Reply:
@brentzundel Evidence is not there to engender trust in the issuer. Trust in the issuer is a necessary pre-cursor to relying on the Evidence. Evidence is there so that the verifier can determine the level of trust to place in the attributes of the subject. The more evidence the subject provided to the issuer, the higher the trust the verifier can have in the subject's claims (obviously up to the maximum trust that the verifier has in the issuer). So the trust in a subject is always LE the trust in the issuer.

Reply:
+1 @David-Chadwick 

@brentzundel in the professional licensing and credentialing space, the hiring body (hospital, for example) will look at the body of work the subject/holder has done as they exercise judgement around fitness of the applicant.  Having an option to put additional information in a credential from the issuer solves the challenge that holders lie and commit fraud in the job hunt all the time.  Evidence allows the verifier to see a more complete picture of the applicant, it has a clear chain of custody and trust.  Selective disclosure is not always the objective of the ecosystem.  We also want a trustworthy and standard way to share details that add nuance to the credential.